scripts/deploy.sh

```diff
@@ -31,7 +31,7 @@ kubectl apply -f manifests/vizier/ui
 kubectl apply -f manifests/vizier/suggestion/random
 kubectl apply -f manifests/vizier/suggestion/grid
 kubectl apply -f manifests/vizier/suggestion/hyperband
-kubectl apply -f manifests/vizier/suggestion/bayesianoptimization
+kubectl apply -f manifests/vizier/suggestion/katib_suggestion
```

Comment:
Why do we rename folder only for BayesianOptimization?

Reply:
Now that multiple algorithms (some of which aren't BO) sit inside a single image we start renaming bayesianoptimization to katib_suggestion.